### PR TITLE
chore: valantis module deployment + Pauser sm

### PR DESCRIPTION
**Pull request by Gevarist**

### CODE REVIEW

File: src/Pauser.sol [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import {EnumerableSet} from
+    "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+import {IPausable} from "./interfaces/IPausable.sol";
+
+import {Ownable} from "@solady/contracts/auth/Ownable.sol";
+
+contract Pauser is IPauser, Ownable {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    EnumerableSet.AddressSet internal _pausers;
```
Comment (kassandraoftroy) [inline src/Pauser.sol:15]: for pure convenience is it worth exposing pausers() as public ?

File: src/Pauser.sol [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+import {IPauser} from "./interfaces/IPauser.sol";
+
+import {EnumerableSet} from
+    "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+import {IPausable} from "./interfaces/IPausable.sol";
+
+import {Ownable} from "@solady/contracts/auth/Ownable.sol";
+
+contract Pauser is IPauser, Ownable {
```
Comment (kassandraoftroy) [inline src/Pauser.sol:12]: Isn't most pausability that we have "onlyGuardian" ? So does this mean Pauser.sol will become the arrakis Guardian address everywhere ? For now guardian was... multisig?

File: src/Pauser.sol (hunk 1 above)
Comment (Gevarist) [reply src/Pauser.sol:15]: recursive type cannot be public

File: src/Pauser.sol (hunk 2 above)
Comment (Gevarist) [reply src/Pauser.sol:12]: pauser inside the guardian was our multisig, but now this contract will become pauser of guardian. And our multisig will become on of the addresses that will be able to call pause of modules.

**Pull request merged**